gitman/git.py

```diff
@@ -113,11 +113,17 @@ def valid():
         return False
 
     log.debug("Checking for a valid git top level...")
-    gittoplevel = git('rev-parse', '--show-toplevel', _show=False)
-    currentdir = pwd(_show=False)
+    gittoplevel = os.path.normpath(
+        os.path.normcase(git('rev-parse', '--show-toplevel', _show=False)[0])
+    )
 
+    currentdir = os.path.normpath(os.path.normcase(pwd(_show=False)))
     status = False
-    if gittoplevel[0] == currentdir:
+
+    log.debug("gittoplevel = %s", gittoplevel)
+    log.debug("currentdir = %s", currentdir)
```

Comment:
I think this is unnecessary since logging previously only happened on failure? @jacebrowning  thoughts/preferences?

Reply:
Yeah, I think this can be removed since the `log.debug()` below should help diagnose failure conditions.

Please note that `gittoplevel` is expected to be a list below.